tests/config.rs

```diff
@@ -0,0 +1,62 @@
+use assert_cmd::prelude::*;
+use std::env;
+use std::fs;
+use std::fs::File;
+use std::io::prelude::*;
+use std::process::{Child, Command, Stdio};
+
+#[test]
+fn it_generates_the_config() {
+    let fake_home_dir = env::current_dir()
+        .expect("could not retrieve cwd")
+        .join(".it_generates_the_config");
+    let cmd = config_with_home(fake_home_dir.to_str().unwrap());
+    let mut stdin = cmd.stdin.unwrap();
+
+    write!(stdin, "a\n").unwrap(); // email
+    write!(stdin, "b\n").unwrap(); // api_key
+
+    let mut buffer = "".to_string();
+    let mut stdout = cmd.stdout.unwrap();
+    stdout
+        .read_to_string(&mut buffer)
+        .expect("could not read output");
+    assert!(buffer.contains("Enter email: \nEnter api key: \n Successfully configured."));
+
+    let config_file = fake_home_dir
+        .join(".wrangler")
+        .join("config")
+        .join("default.toml");
+
+    let config = fs::read_to_string(&config_file).expect("could not read config");
+    assert_eq!(
+        config,
+        r#"email = "a"
+api_key = "b"
+"#
+    );
+
+    // check dir permissions (but not on windows)
+    if !cfg!(target_os = "windows") {
```

Comment:
What do we do on Windows?

Reply:
I don't know, cc @steveklabnik @ashleygwilliams?